eng/pipelines/runtimelab/install-llvm.ps1

```diff
@@ -1,3 +1,7 @@
+param (
+    [string]$buildConfig = "Release"
```

Comment:
Do we want this defaulted?

Seems it'd be easier to debug bad parameter failures if its required (but it's not a big deal).

Reply:
Removed.